code/src/components/News/Album/Album.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const Album = (props) => {
+  const artistsList = props.artistName.map((element) => {
+    return (<span>{element.name} </span>)
+  })
+  return (
+    <article className="album">
+      <img src={props.img} alt="album cover" className="album-img" />
```

Comment:
Since you have the album name as a prop here, I think that would've made a great alt text :)